encoding/ssz/query/analyzer.go

```diff
@@ -144,13 +219,21 @@ func analyzeListType(typ reflect.Type, elementInfo *sszInfo, limit uint64) (*ssz
 
 		fixedSize:  offsetBytes,
 		isVariable: true,
+
+		listInfo: &listInfo{
+			limit:   limit,
+			element: elementInfo,
+			// NOTE: Length is not known until unmarshalling.
+			// This will be set later in `PopulateFromValue`.
```

Comment:
1. What do you mean by `unmarshalling`? Unmarshalling from bytes with `UnmershalSSZ`?
2. I would remove the name of the function from this comment. Oftentimes the name of the function changes in the future and the comment becomes stale